tests/conftest.py

```diff
@@ -5008,49 +5002,43 @@ def profiler_name() -> str:
 
 
 @pytest.fixture(scope="function")
-def store_name() -> str:
+def profiler_store_name() -> str:
     return "profiler_store"
```

Comment:
Big big thank you for this 🙇🏽 appreciate you cleaning it up

Reply:
🙇  😄